sql/moz-fx-data-shared-prod/mozilla_vpn_derived/acquisition_funnel_v1/query.sql

```diff
@@ -0,0 +1,107 @@
+WITH logged_in AS (
+  SELECT
+    fxa_uid,
+    MIN(created_at) AS first_logged_in,
+  FROM
+    mozilla_vpn_external.users_v1
+  GROUP BY
+    fxa_uid
+),
+viewed_login AS (
+  SELECT
+    fxa_uid,
+    MIN(flow_started) AS first_viewed_login,
+  FROM
+    login_flows_v1
+  FULL JOIN
+    logged_in
+  USING
+    (fxa_uid)
+  WHERE
+    -- include entries in users_v1 that are not present in login_flows_v1
+    first_logged_in IS NULL
+    -- TODO: exclude login on desktop client, and second login attempt on mobile.
+    -- As an approximation, filter out flows that started after first login.
+    OR first_logged_in > flow_started
+  GROUP BY
+    fxa_uid,
+    -- when fxa_uid is missing, produce one row per flow_id
```

Comment:
This will undercount our conversion rates, since one person can be responsible for many `flow_id`s but IIUC at most one of those would be associated with any conversion? But such is life

Reply:
> This will undercount our conversion rates, since one person can be responsible for many `flow_id`s but IIUC at most one of those would be associated with any conversion? But such is life

all of the flows that have a given fxa_uid are counted as a single conversion, yeah. I don't think we have anything longer lived than flow id that we could use to reduce that overcounting.